Manager.cfc

```diff
@@ -0,0 +1,103 @@
+component {
+
+	/*
+		relevant javadoc:
+			- http://www.ehcache.org/apidocs/2.10.4/net/sf/ehcache/Cache.html
+			- http://www.ehcache.org/apidocs/2.10.4/net/sf/ehcache/CacheManager.html
+	*/
+
+	Manager function init() {
+    	return this;
+	}
+
+	Cache function addCache(required string name, string copyFrom) {
+		if(!getInstance().cacheExists(arguments.name)) {
+			if(structKeyExists(arguments, "copyFrom")) {
+				local.config = getInstance().getCache(arguments.copyFrom).getCacheConfiguration().clone();
+
+				local.config.name(arguments.name);
+
+				getInstance().addCacheIfAbsent(createObject("java", "net.sf.ehcache.Cache").init(local.config));
+			} else {
+                getInstance().addCacheIfAbsent(arguments.name);
+			}
+		}
+
+		return getCache(arguments.name);
+	}
+
+	boolean function cacheExists(required string name, boolean deepSearch = false) {
+		if(arguments.deepSearch) {
+			for(local.managerName in getManagerNames()) {
+				if(getInstance(local.managerName).cacheExists(arguments.name)) {
+					return true;
+				}
+			}
+		}
+
+		return getInstance().cacheExists(arguments.name);
+	}
+
+	Manager function createFromPath(required string path) {
+		return new Manager().setInstance(createObject("java", "net.sf.ehcache.CacheManager").newInstance(arguments.path));
+	}
+
+	Manager function createFromXML(required string xml) {
+		local.is = createObject("java", "java.io.ByteArrayInputStream").init(arguments.xml.getBytes());
```

Comment:
@timmixell I could go either way on this one. We allow acronyms, but this also looks like "is" so ... :shurg: